akka-http/src/main/scala/akka/http/javadsl/server/directives/FutureDirectives.scala

```diff
@@ -30,6 +30,16 @@ abstract class FutureDirectives extends FormFieldDirectives {
   }
 
   /**
+   * "Unwraps" a `CompletionStage<T>` and runs the inner route after future
+   * completion with the future's value as an extraction of type `Try<T>`.
+   */
+  def onComplete[T](cs: CompletionStage[T], inner: JFunction[Try[T], Route]) = RouteAdapter {
```

Comment:
Why is that? JavaDoc doesn't support it? Just saw it below on `onCompleteWithBreaker`.